app/assets/stylesheets/observations/identify/observation_modal.scss

```diff
@@ -554,6 +554,15 @@ body > [role="dialog"] .ObservationModal {
             top: 1px;
           }
         }
+        img.app-thumbnail {
+          position: absolute;
+          left: 0;
+          top: 0;
```

Comment:
Can you add 1px here, and do the same for the `fa-cloud-upload` version too? That looks a bit more vertically aligned to me.